user-guide/upgrade-to-edge-stack.md

```diff
@@ -61,11 +61,19 @@ kubectl delete service -n <namespace> test-aes
 Apply the new CRDs, resources and restart the Ambassador Edge Stack pod for changes to take effect:
 
 ```
-kubectl apply -n <namespace> -f https://www.getambassador.io/early-access/yaml/aes-crds.yaml && \
-kubectl apply -n <namespace> -f https://www.getambassador.io/early-access/yaml/resources-migration.yaml && \
+kubectl apply -n <namespace> -f https://www.getambassador.io/yaml/aes-crds.yaml && \
+kubectl apply -n <namespace> -f https://www.getambassador.io/yaml/resources-migration.yaml && \
 kubectl rollout restart deployment/aes
 ```
 
-## 6. What's Next?
+## 6. Access the Edge Policy Console
+
+You can now access the Edge Policy Console with the following options:
+* `edgectl login -n <namespace> <AES_host>` or
+* `https://{{AES_URL}}/edge_stack/admin`
+
+If you log in with the URL, it allows you to have a `Mapping` at `prefix: /` if desired.
```

Comment:
Sorry but I think this point just kind of confuses the message. 

They should just use the `/edge_stack/admin` path since that will always work. No need to introduce more information about why this works or not. I would leave this out but I will default to your judgement 🙂